src/main/java/dk/bec/polonez/reservationsystem/model/Offer.java

```diff
@@ -25,6 +25,10 @@
     @JoinColumn
     private User owner;
 
+    private double pricePerUnit;
+
+    private String unitDuration;
```

Comment:
**advice** I can see this field means some duration (okres trwania) - is it possible to use ```Duration``` type from ```java.time```?